Gopkg.toml

```diff
@@ -37,9 +10,13 @@
   name = "github.com/jdkato/prose"
   version = "1.1.0"
 
-[[override]]
+[[constraint]]
+  name = "github.com/pkg/errors"
+  version = "0.8.0"
+
+[[constraint]]
   name = "github.com/sergi/go-diff"
-  revision = "da645544ed44df016359bd4c0e3dc60ee3a0da43"
+  version = "1.0.0"
```

Comment:
The hash must be exactly that

Reply:
this is a bit weird ... is there anyway `go-diff` can add a new tag for this instead?